### PR TITLE
Fix up 'serviceacccount' example

### PR DESCRIPTION
**Pull request by sbose78**
# Changes

Update example to use .spec.taskSpec.steps

/kind documentation

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [ ] Release notes block has been filled in or deleted (only if no user facing changes)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md) for more details._

Double check this list of stuff that's easy to miss:

- If you are adding [a new binary/image to the `cmd` dir](../cmd), please update
  [the release Task](../tekton/publish.yaml) to build and release this image.

## Reviewer Notes

If [API changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md) are included, [additive changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#additive-changes) must be approved by at least two [OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS) and [backwards incompatible changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-incompatible-changes) must be approved by [more than 50% of the OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS), and they must first be added [in a backwards compatible way](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-compatible-changes-first).

(https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-compatible-changes-first).

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Review by jerop — APPROVED** [review]
thanks @sbose78

File: docs/taskruns.md [hunk 1]
```diff
@@ -678,12 +678,14 @@ spec:
   resources:
     inputs:
       - name: workspace
-        type: git
-  steps:
-    - name: config
-      image: ubuntu
-      command: ["/bin/bash"]
-      args: ["-c", "cat README.md"]
+        resourceSpec:
```
Comment (sbwsg) [inline docs/taskruns.md:681]: This needs a little bit more to be a successful taskrun:

```yaml
        resourceSpec:
          type: git
          params:
            - name: url
              value: https://github.com/tektoncd/pipeline.git
```

And the `taskSpec` needs the following in addition to the steps:

```yaml
  taskSpec:
    resources:
      inputs:
      - name: workspace
        type: git
```

**Comment by pritidesai** [comment]
thanks @sbose78 please include the suggested changes so that we can merge this 🙏

**Comment by sbose78** [comment]
Sure, doing it asap!

**Comment by jerop** [comment]
pulled in this commit and updated it in https://github.com/tektoncd/pipeline/pull/4274

thanks @sbose78!

**Pull request closed without merging**